Http.Mqtt.Connector.Svc/DependencyExtensions.cs

```diff
@@ -105,4 +99,26 @@ but it may clash with topic names which have underscores naturally.
 
         return services;
     }
+
+    private static string SanitizeTopicName(string baseUrl, string relativeUrl, string baseTopic)
+    {
+        var originalUrl = new Uri(new Uri(baseUrl), relativeUrl).ToString();
+        var hash = ComputeSha256Hash(originalUrl);
+        var sanitizedUrl = originalUrl.Replace(".", "_", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Ideally, we should make it extensible apart from the absolute required ones (i.e. add '#' and '+' in code,  as default) . Extensibility here means user can define additional replacement chars (KV pair) in appsettings.json; this is useful if users want to define themselves the additional replacements for their env. One example here is, when this connector runs in AIO env, AIO currently does not support (in UI) dots(.) but they perfectly valid from MQTT perspective.